website/docs/r/kms_key_policy.html.markdown

```diff
@@ -0,0 +1,89 @@
+---
+subcategory: "Key Management Service"
+layout: "ibm"
+page_title: "IBM : kms-key-policies"
+description: |-
+  Manages key policies for IBM hs-crypto and kms.
+---
+
+# ibm\_kms_key_policies
+
+Provides a resource to manage key policies for hs-crypto and key-protect services. This allows key policies to be created. Key policies can be created for an existing kms key resource.
+
+
+## Example usage to create a Key and associate a key policy.
+
+```terraform
+resource "ibm_resource_instance" "kms_instance" {
+  name     = "instance-name"
+  service  = "kms"
+  plan     = "tiered-pricing"
+  location = "us-south"
+}
+
+resource "ibm_kms_key" "key" {
+  instance_id = ibm_resource_instance.kp_instance.guid
+  key_name       = "key"
+  standard_key   = false
+}
+
+resource "ibm_kms_key_policies" "keyPolicy" {
+  instance_id = ibm_resource_instance.kms_instance.guid
+  key_id = ibm_kms_key.key.key_id
+     rotation {
+       interval_month = 3
+    }
+    dual_auth_delete {
+       enabled = false
+     }
+}
+```
+
+**NOTE**
+1) `terraform destroy` does not remove the policies of the Key but only clears the state file. Key Policies get deleted when the associated key resource is destroyed.
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `instance_id` - (Required, Forces new resource, string) The hs-crypto or key-protect instance guid.
+* `endpoint_type` - (Optional, Forces new resource, string) The type of the endpoint (public or private) to be used for creating keys.
+* `rotation` - (Optional, list) Specifies the key rotation time interval in months, with a minimum of 1, and a maximum of 12.
+  * `interval_month` - (Required, int) Specifies the key rotation time interval in months.
+      **CONSTRAINTS**: 1 ≤ value ≤ 12
+      **NOTE**: Rotation policy is cannot be set for standard key and imported key. Once Rotation Policy is set, it is not possible to unset/remove it using Terraform.
+* `dual_auth_delete` - (Required, list) Data associated with the dual authorization delete policy.
+  * `enabled` - (Optional, bool) If set to true, Key Protect enables a dual authorization policy on a single key.
+    **NOTE**: Once the dual authorization policy is set on the key, it cannot be reverted. A key with dual authorization policy enabled cannot be destroyed using Terraform.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - Unique Identifier for the terraform resource.
```

Comment:
Type of attribute is not mentioned here..
Please follow according to kms_key docs